### ISSUE TITLE
Add Wheezy-backprots repo

### ISSUE THREAD

**Issue by danny-cheung** [siso/chef-takeaway#14]
Needed for the utilities such as jq.

**Comment by danny-cheung** [comment]
/etc/apt/sources.list.d/backports.list:

```
# wheezy-backports
deb http://ftp.au.debian.org/debian wheezy-backports main
deb-src http://ftp.au.debian.org/debian wheezy-backports main
```

**Comment by siso** [comment]
Fixed with 0.3.0
